src/_data/catalog/widgets.json

```diff
@@ -1288,6 +1497,17 @@
     "link": "https://api.flutter.dev/flutter/widgets/PositionedTransition-class.html",
     "image": "<img alt='' src='/assets/images/docs/catalog-widget-placeholder.png'>"
   },
+  {
+    "name": "Progress indicator",
+    "description": "Progress indicators inform users about the status of ongoing processes, such as loading an app or submitting a form.",
```

Comment:
**issues:**
- Make consistent with other descriptions.
- Remove the circular definition.
```suggestion
    "description": "Vertical line that changes color as an ongoing process, such as loading an app or submitting a form, completes.",
```